### ISSUE TITLE
Can you provide the version information of the environment, please?

### ISSUE THREAD

**Issue by shannon-paul** [vivian1993/NCRL#1]
I'm reproducing the experiment with many different errors due to the version of pytorch, can you provide me with the version information of your environment at the time of the experiment, for example a file like requirement.txt? Thanks :)

**Comment by rabi-fei** [comment]
A similar error happened to me. Warning: one of the variables needed for gradient computation has been modified by an inplace operation.
And I find that it was caused by unstable softmax algorithm. Then I fixed the error by replacing line 65 in model.py with the following stable code. Hoping this also helps you!
            v = torch.exp(scores - scores.max(dim=-1)[0].unsqueeze(-1))
            probs = v / v.sum(dim=-1).unsqueeze(-1)
By the way, my pytorch version is 1.10.2.

**Comment by shannon-paul** [comment]
> 类似的错误也发生在我身上。警告：梯度计算所需的一个变量被替换操作修改了。 而我发现这是由不稳定的softmax算法造成的。然后我用下面的稳定代码替换了model.py中的第65行，修复了这个错误。希望这也能帮助你 v = torch.exp(scores - scores.max(dim=-1)[0].unsqueeze(-1)) probs = v / v.sum(dim=-1).unsqueeze(-1) 顺便说一下，我的pytorch版本是1.10.2。

Thank you for your help :)👍
